assets/src/components/SelectCard.js

```diff
@@ -117,23 +136,54 @@ const SelectCard = props => {
 
   function getCardImage (cardData) {
     if (cardData && cardData.image) {
-      return <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+      return (
+        <>
+          <Link className={classes.viewLink} tabIndex={-1} to={cardData.path}>
+            <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+          </Link>
+        </>
+      )
     } else {
       return null
     }
   }
 
+  function getHelpLink (cardData) {
+    if (cardData.helpUrl) {
+      return (
+        <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
+          <Tooltip title={'About ' + cardData.title}>
+            <MUILink className={classes.infoLink} href={cardData.helpUrl} target='_blank' rel='noopener noreferrer'>
+              <InfoIcon />
+            </MUILink>
+          </Tooltip>
+        </Typography>
+      )
+    }
+  }
+
   function getLinkContents (cardData) {
     const cardImage = getCardImage(cardData)
 
     const cardContent = (
       <CardContent className={classes.content}>
-        <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
-          {cardData.title}
-        </Typography>
-        <Typography component='p' className={classes.description}>
-          {cardData.description}
-        </Typography>
+        <Grid container className={classes.titleArea}>
+          <Grid item xs={11}>
+            <Link tabIndex={-1} to={cardData.path} className={classes.viewLink}>
```

Comment:
Can the `Link` be wrapped around the `Grid container`, instead of around each individual `Grid item`?

Reply:
There may be a better way to do it, but the `Grid container` has both the link to the view and the link to help, so I had to come up with a way to wrap the individual areas that should link to different places.  This was what I came up with.

I imagine there would be a way to have a relatively positioned overlay over the info icon with a higher z-index and the help link, perhaps, but this is what I came up with.

Reply:
Mmm, I see. Didn't think about how this would affect that other item. What you have looks good for now and works!